umm-lib/test/cmr/umm/test/echo10/granule.clj

```diff
@@ -445,4 +446,18 @@
             "Exception while parsing invalid XML: Line 3 - cvc-type.3.1.3: The value 'XXXX-01-05T05:30:30.550-05:00' of element 'InsertTime' is not valid."
             "Exception while parsing invalid XML: Line 4 - cvc-datatype-valid.1.2.1: 'XXXX-01-05T05:30:30.550-05:00' is not a valid value for 'dateTime'."
             "Exception while parsing invalid XML: Line 4 - cvc-type.3.1.3: The value 'XXXX-01-05T05:30:30.550-05:00' of element 'LastUpdate' is not valid."]
-           (g/validate-xml (s/replace valid-granule-xml-w-sn-ver "2010" "XXXX"))))))
+           (g/validate-xml (string/replace valid-granule-xml-w-sn-ver "2010" "XXXX"))))))
+
+(deftest generate-data-granule-test
+  (testing "Testing the generate-data-granule method when the size doesn't exist - making sure that
+            a null pointer exception isn't thrown. CMR-7605"
+    (let [granule-data (g/generate-data-granule
+                         (umm-g/map->DataGranule
+                           {:size-in-bytes 71938553
+                            :size-unit "KB"
+                            :producer-gran-id "0000000.0000001.hdf"
+                            :day-night "NIGHT"}))]
+      (is (and (= (cx/string-at-path granule-data [:DataGranuleSizeInBytes])
```

Comment:
Change this line to two separate `is` verification rather than an `and`.

Reply:
I wanted them to be anded together to signify that both have to be true, but making it two separate is accomplishes the same thing.

Reply:
fixed.